onnx/common/stl_backports.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
```

Comment:
It's probably copied from caffe2? folly later added an additional android check.
Besides, I recommend don't flip the if-else from the original file, especially when there are several conditional checks.

Reply:
It was a slightly modified copy of the caffe2 implementation. I now switched to the one from folly.